src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -39,6 +45,29 @@ interface IEigenPodManager is IPausable {
         bytes32 withdrawalRoot
     );
 
+    //info for each withdrawal called back by proof service
+    struct WithdrawalCallbackInfo {
+        // the address of the pod owner
+        address podOwner;
+        // lower bound of the withdrawal period
+        uint64 startTimestamp;
+        // upper bound of the withdrawal period
+        uint64 endTimestamp;
+        // amount being proven for withdrawal
+        uint256 provenPartialWithdrawalSumWei;
+        // prover fee
+        uint256 fee;
+        // user-signed maxFee
```

Comment:
Do we not want to validate this signature in the contracts? If the proof service is compromised they can make this up entirely